SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -30,69 +34,66 @@ bool WriteFlagSection(TSharedRef<TJsonWriter<>> Writer, const FString& Key, cons
 	return true;
 }
 
-bool WriteWorkerSection(TSharedRef<TJsonWriter<>> Writer, const FWorkerTypeLaunchSection& Worker)
+bool WriteWorkerSection(TSharedRef<TJsonWriter<>> Writer, const FName& WorkerTypeName, const FWorkerTypeLaunchSection& WorkerConfig)
 {
 	Writer->WriteObjectStart();
-		Writer->WriteValue(TEXT("worker_type"), *Worker.WorkerTypeName.ToString());
+		Writer->WriteValue(TEXT("worker_type"), *WorkerTypeName.ToString());
 		Writer->WriteArrayStart(TEXT("flags"));
-			for (const auto& Flag : Worker.Flags)
+			for (const auto& Flag : WorkerConfig.Flags)
 			{
 				WriteFlagSection(Writer, Flag.Key, Flag.Value);
 			}
 		Writer->WriteArrayEnd();
 		Writer->WriteArrayStart(TEXT("permissions"));
 			Writer->WriteObjectStart();
-			if (Worker.WorkerPermissions.bAllPermissions)
-			{
-				Writer->WriteObjectStart(TEXT("all"));
-				Writer->WriteObjectEnd();
-			}
-			else
-			{
-				Writer->WriteObjectStart(TEXT("entity_creation"));
-					Writer->WriteValue(TEXT("allow"), Worker.WorkerPermissions.bAllowEntityCreation);
-				Writer->WriteObjectEnd();
-				Writer->WriteObjectStart(TEXT("entity_deletion"));
-					Writer->WriteValue(TEXT("allow"), Worker.WorkerPermissions.bAllowEntityDeletion);
-				Writer->WriteObjectEnd();
-				Writer->WriteObjectStart(TEXT("entity_query"));
-					Writer->WriteValue(TEXT("allow"), Worker.WorkerPermissions.bAllowEntityQuery);
+				if (WorkerConfig.WorkerPermissions.bAllPermissions)
+				{
+					Writer->WriteObjectStart(TEXT("all"));
+					Writer->WriteObjectEnd();
+				}
+				else
+				{
+					Writer->WriteObjectStart(TEXT("entity_creation"));
+						Writer->WriteValue(TEXT("allow"), WorkerConfig.WorkerPermissions.bAllowEntityCreation);
+					Writer->WriteObjectEnd();
+					Writer->WriteObjectStart(TEXT("entity_deletion"));
+						Writer->WriteValue(TEXT("allow"), WorkerConfig.WorkerPermissions.bAllowEntityDeletion);
+					Writer->WriteObjectEnd();
+					Writer->WriteObjectStart(TEXT("entity_query"));
+						Writer->WriteValue(TEXT("allow"), WorkerConfig.WorkerPermissions.bAllowEntityQuery);
 					Writer->WriteArrayStart("components");
-					for (const FString& Component : Worker.WorkerPermissions.Components)
+					for (const FString& Component : WorkerConfig.WorkerPermissions.Components)
 					{
 						Writer->WriteValue(Component);
 					}
 					Writer->WriteArrayEnd();
-				Writer->WriteObjectEnd();
-			}
+					Writer->WriteObjectEnd();
+				}
 			Writer->WriteObjectEnd();
 		Writer->WriteArrayEnd();
-		if (Worker.MaxConnectionCapacityLimit > 0)
+		if (WorkerConfig.MaxConnectionCapacityLimit > 0)
 		{
 			Writer->WriteObjectStart(TEXT("connection_capacity_limit"));
-				Writer->WriteValue(TEXT("max_capacity"), Worker.MaxConnectionCapacityLimit);
+				Writer->WriteValue(TEXT("max_capacity"), WorkerConfig.MaxConnectionCapacityLimit);
```

Comment:
```suggestion
			Writer->WriteValue(TEXT("max_capacity"), WorkerConfig.MaxConnectionCapacityLimit);
```

Reply:
Apparently the convention going on in this file is to indent the lines between ObjectStart and ObjectEnd, why make an exception here ?